src/base.jl

```diff
@@ -18,6 +18,7 @@ const SYSTEM_KWARGS = Set((
     :time_series_read_only,
     :timeseries_metadata_file,
     :unit_system,
+    :validate,
```

Comment:
How does a user distinguish between validate and run checks? If I set run checks to false and validate to true was to that mean?

Reply:
We also need to update the docstring

Reply:
fair... I was just directly passing the pm kwarg... I'll make it more clear

Reply:
FWIW I don't think that validate is correct in PM either since it doesn't validate really but mutates the underlying data.